pivy-importer/src/main/groovy/com/linkedin/python/importer/ivy/IvyFileWriter.groovy

```diff
@@ -24,7 +24,7 @@ import org.apache.commons.io.FilenameUtils
 
 @TupleConstructor
 class IvyFileWriter {
-    final String name
+    String name
```

Comment:
I always get nervous when I see "final" removed.  Marking something "final" is making a strong promise/guarantee in the code, and removing that guarantee means you have to be sure nobody was taking advantage of it.  Was that done?

Reply:
We can add it back, we used to fix the name, since user might input name which has wrong case or using "-" to replace "_",  we will help them to correct the name, now this will be done earlier, so we can get "final" back here.